unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/EmotesHUD/EmotesHUDController.cs

```diff
@@ -88,20 +67,8 @@ public EmotesHUDController(UserProfile userProfile, BaseDictionary<string, Weara
             emoteAnimations.OnAdded += OnAnimationAdded;
 
             ConfigureShortcuts();
-        }
 
-        // TODO (Santi): Remove it when we don't longer need keep the retrocompatibility.
-        private void LoadDefaultEmotesIfNeeded()
-        {
-            if (!isEmotesCustomizationFFEnabled)
-            {
-                List<EquippedEmoteData> storedEquippedEmotesData = new List<EquippedEmoteData>();
-                foreach (string emoteId in oldEmotes)
-                {
-                    storedEquippedEmotesData.Add(new EquippedEmoteData { id = emoteId, cachedThumbnail = null });
-                }
-                emotesCustomizationDataStore.equippedEmotes.Set(storedEquippedEmotesData);
-            }
+            emotesCustomizationDataStore.isWheelInitialized.Set(true);
```

Comment:
I find a bit strange that the `isWheelinitialized` state is set by the `EmotesHUDController` and not the Wheel itself.

The whell should be a complely separated system, it just reacts to changes in the `equippedEmotes` collection to update the info, it shouldnt know anybody and it shouldnt be known by anybody either.

Reply:
The EmotesHUDController is the "Wheel". I conserved the same name as the old wheel had... Maybe I should change this name by something like `EmotesWheelController` to avoid confussions?

Reply:
Yes pls! I thought this was the controller in the backpack!